index.html

```diff
@@ -114,6 +114,37 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Simrat Karamjeet</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/simratkaramjeet" target="_blank">simratkaramjeet</a>
+          </p>
+          <p class="about">I am a software developer and I also enjoy travelling, herbal medicine, watercolor painting, stock market and rock climbing.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://github.com/Syknapse/Contribute-To-This-Project" target="_blank" title="First Resource">
+                  First open-source contribution
+                </a>
+              </li>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/" target="_blank" title="Second Resource">
+                  Useful reference for languages
+                </a>
+              </li>
+              <li>
+                <a href="https://stackoverflow.com/" target="_blank" title="Third resource">
+                  Best reference for debugging
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
This `*END TEMPLATE*` should also be changed. 
It will be easier to identify each contributor's code afterwards. Otherwise, some might even mistake your code for the original template.